app/utils.py

```diff
@@ -522,3 +522,28 @@ async def post_webform(request, form_data):
                                         f'{rhsvc_url}/webform',
                                         auth=request.app['RHSVC_AUTH'],
                                         request_json=form_json)
+
+    @staticmethod
+    async def register_new_case(request, data):
+        new_case_json = {
+            'collectionExerciseId': '03f58cb5-9af4-4d40-9d60-c124c5bddf09',  # Dummy value
+            'schoolId': '1',  # Dummy value
+            'schoolName': data['school_name'],
+            'consentGivenTest': True,  # Fixed value
+            'consentGivenSurvey': True,  # Fixed value
+            'firstName': data['parent_first_name'],
+            # Unable to submit parent_middle_names
+            'lastName': data['parent_last_name'],
+            'childFirstName': data['child_first_name'],
+            'childMiddleNames': data['child_middle_names'],
+            'childLastName': data['child_last_name'],
+            'childDob': data['child_dob'],
```

Comment:
should we be sending `additionalInfo`  ? which is part of the DTO receiving class